Dockerfile

```diff
@@ -1,17 +1,10 @@
-FROM openjdk:17-slim
-CMD ["/usr/local/openjdk-17/bin/java", "-jar", "/opt/ssdc-rm-job-processor.jar"]
+FROM eclipse-temurin:17-jdk-alpine
 
+ARG JAR_FIL=ssdc-rm-job-processo*.jar
+CMD ["/opt/java/openjdk/bin/java", "-jar", "/opt/ssdc-rm-job-processor.jar"]
```

Comment:
Could this be simplified to just 
```dockerfile
CMD ["java", "-jar", "/opt/ssdc-rm-job-processor.jar"]
```
? 

I'm not sure why we were specifying the explicit path to `java` before, but I don't think we need to carry it on.

Reply:
yeah fair point